src/r5py/r5/base_travel_time_matrix_computer.py

```diff
@@ -194,7 +194,7 @@ def origins(self, origins):
 
     def _travel_times_per_origin(self, from_id):
         request = copy.copy(self.request)
-        request.origin = self.origins[self.origins.id == from_id].geometry
+        request.origin = self.origins.loc[self.origins.id == from_id, "geometry"].item()
```

Comment:
What's this addition about?

Reply:
New pandas version is warning of looming deprecation -> does not pass tests (we fail on warnings, and catching this tells me that’s a good idea)